atom/browser/api/atom_api_web_contents.cc

```diff
@@ -949,7 +951,23 @@ bool WebContents::HandleContextMenu(const content::ContextMenuParams& params) {
     Emit("pepper-context-menu", std::make_pair(params, web_contents()));
     web_contents()->NotifyContextMenuClosed(params.custom_context);
   } else {
-    Emit("context-menu", std::make_pair(params, web_contents()));
+    // For forgetting custom dictionary option
+    content::ContextMenuParams new_params(params);
+    if (new_params.misspelled_word.empty()) {
+      auto browser_context = web_contents()->GetBrowserContext();
+      if (browser_context) {
+        SpellcheckService* spellcheck =
+          SpellcheckServiceFactory::GetForContext(browser_context);
+        if (spellcheck) {
+          if (spellcheck->GetCustomDictionary()
+              ->HasWord(base::UTF16ToUTF8(new_params.selection_text))) {
+            new_params.misspelled_word =
```

Comment:
It is used for https://github.com/brave/browser-laptop/pull/9951/files#diff-b9f459fd47114771333b78b87fac7daaR557 which provides user an option to remove added word to custom dictionary.
because if a word is in custom dictionary, we can't tell if it is correctly spelled word or in custom dictionary from `ContextMenuParams`.  `misspelled_word` and `dictionary_suggestions` will be empty